dbaas/logical/models.py

```diff
@@ -219,7 +219,12 @@ def update_task(self, task):
             lock = DatabaseLock.objects.select_for_update().filter(
                 database=self
             ).first()
-            if lock.task.task_name != task.task_name or not lock.task.is_status_error:
+
+            task_name = task.task_name
+            if task_name.endswith('_rollback'):
+                task_name = task_name.rsplit('_rollback', 1)[0]
```

Comment:
U think one of than is better ? 
1 - task_name = task_name[:task_name.rfind('_rollback')]
2 - task_name = task_name.replace("_rollback")